src/pages/[platform]/build-a-backend/graphqlapi/connect-api-to-existing-database/index.mdx

```diff
@@ -0,0 +1,670 @@
+        }
+      }
+    }
+  ),
+  authorizationModes: {
+    apiKeyConfig: { expires: Duration.days(7) }
+  },
+  translationBehavior: {
+    sandboxModeEnabled: true
+  }
+});
+```
+
+Before deploying:
+
+- Change the `dbType` to match your database engine of either MySQL or PostgreSQL.
+
+- Update the `vpcId`, `securityGroupIds`, and `subnetAvailabilityZoneConfig` according to your AWS environment.
+
+- Update the SSM parameter paths within `dbConnectionConfig`.
+
+</Block>
+</BlockSwitcher>
+
+<Accordion title="RDS Proxy for improved connectivity" headingLevel="3" eyebrow="Learn more">
+
+Consider adding an RDS Proxy in front of the cluster to manage database connections.
+
+When using AWS AppSync with a relational database like Amazon RDS, each query from your application needs to open a separate connection to the database.
+
+If there are a large number of queries occurring concurrently, it can exceed the connection limit on the database and result in errors like "Too many connections". To avoid this, the Amplify CLI can use an RDS Proxy when connecting your AppSync API to a database.
+
+The RDS Proxy acts as an intermediary sitting in front of your database. Instead of each application query opening a direct connection to the database, they will connect through the Proxy. The Proxy helps manage and pool these connections to avoid overwhelming your database cluster. This improves the availability of your API, allowing more queries to execute concurrently without hitting connection limits.
+
+However, there is a tradeoff of increased latency - queries may take slightly longer as they wait for an available connection from the Proxy pool. There are also additional costs associated with using RDS Proxy. Please refer to the [pricing page for RDS Proxy](https://aws.amazon.com/rds/proxy/pricing/) to learn more.
+
+</Accordion>
+
+## Apply authorization rules on the models
+
+The `@auth` directive can be used to restrict access to data and operations by specifying authorization rules. It allows granular access control over the GraphQL API based on the user's identity and attributes.
+
+All model level authorization rules are supported for Amplify GraphQL schemas generated from MySQL and PostgreSQL databases.
+
+<Callout>**NOTE:** Field level auth rules are not supported.</Callout>
+
+In the example below, public users authorized via API Key are granted unrestricted access to all posts.
+
+Add the following auth rule to the `Post` model within the `schema.sql.graphql` file:
+
+```graphql
+type Post @model @auth(rules: [{ allow: public }]) {
+  id: String! @primaryKey
+  title: String!
+  content: String!
+}
+```
+
+In a real world scenario, you can instead define auth rules that only allow public users to read posts, and authenticated users the ability to update or delete their posts.
+
+For more information on each rule please refer to our documentation on [Authorization rules](/cli/graphql/authorization-rules/).
```

Comment:
the URL will be different. make sure it's correct

Reply:
Corrected the link